Models/Http/Response.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Infrastructure\Models\Http;
+
+
+use Psr\Http\Message\StreamInterface;
+
+class Response
+{
+    /**
+     * @var \GuzzleHttp\Psr7\Response
+     */
+    private $guzzleResponse;
```

Comment:
didn't get, why do you use GuzzleResponse?

Reply:
same